seml/database.py

```diff
@@ -63,10 +63,8 @@ def get_mongodb_config(path=SETTINGS.DATABASE.MONGODB_CONFIG_PATH):
     """
 
     access_dict = {}
-
-    error_str = f"Please supply your MongoDB credentials at {path} in the format:\n"\
-                "username: <your_username>\npassword: <your_password>\nport: <port>\n"\
-                "database:<database_name>\n host: <hostname>"
+    error_str = f"MongoDB credentials could not be read at {path}.\n"\
```

Comment:
should we use `seml --configure` or `seml configure`? The latter seems more in line with other commands (queue, start, etc.; similar to e.g. `conda`, `pip`), the former is more common in general.

Reply:
`seml configure` would require another subparser, which would also give us more flexibility.

Reply:
Good idea. See 4b573e0.